salt/modules/acme.py

```diff
@@ -216,6 +217,7 @@ def cert(
         if dns_plugin == "cloudflare":
             cmd.append("--dns-cloudflare")
             cmd.append("--dns-cloudflare-credentials {}".format(dns_plugin_credentials))
+            cmd.append("--dns-cloudflare-propagation-seconds {}".format(dns_plugin_propagate_seconds))
```

Comment:
Please add docs for this new parameter as well as a ``.. version-added::``. If you can, document the other parameters as well.